modules/xfem/doc/content/syntax/XFEM/index.md

```diff
@@ -1 +1,78 @@
-!template load file=stubs/moose_system.md.template name=XFEM syntax=/XFEM
\ No newline at end of file
+# XFEM Action System
+
+## Overview
+
+The XFEM Action is an object required when running MOOSE simulations using XFEM. It provides
+general XFEM solution parameters to the `XFEMApp` class. It has several functions:
```

Comment:
Replace all instances of XFEMApp with XFEM.  XFEMApp doesn't really do much -- just registers syntax.